lib/semantics/expression.cc

```diff
@@ -1470,9 +1474,46 @@ static MaybeExpr AnalyzeExpr(ExpressionAnalysisContext &context,
       }
       unavailable.insert(symbol->name());
       if (MaybeExpr value{AnalyzeExpr(context, expr)}) {
-        // TODO pmk: C7104, C7105 check that pointer components are
-        // being initialized with data/procedure designators appropriately
-        result.Add(*symbol, std::move(*value));
+        bool isNULL{std::holds_alternative<NullPointer>(value->u)};
```

Comment:
I think `isNull` is a better spelling, to be consistent with `NullPointer`.

Reply:
The capitalization was meant to suggest the NULL() intrinsic function, as I've used all-caps for other intrinsic functions in the evaluation library.

Reply:
OK, makes sense.